finality-aleph/src/nodes.rs

```diff
@@ -131,14 +132,12 @@ where
         }
     });
 
-    let (gossip_network_service, authentication_network, block_sync_network) =
-        GossipService::new(network_event_stream, spawn_handle.clone(), registry.clone());
     let gossip_network_task = async move {
```

Comment:
you probablt want to rename gossip_network_task

Reply:
done